_sass/jekyll-theme-haymarket.scss

```diff
@@ -935,7 +935,7 @@ a,
   }
 
   .tile-container-text {
-    height: 2rem;
+    height: 3rem;
```

Comment:
When I ran this locally, this was causing a slight misalignment between tiles with images and overlapping the border that is supposed to show up on focus. I think a quick fix is to boost this up to 3.25rem. 

![Screen Shot 2021-07-10 at 11 32 15 AM](https://user-images.githubusercontent.com/38137359/125170062-b68ac080-e172-11eb-895d-43000d6f0f67.png)

Reply:
Hey @nickharriscodes, just wondering what settings you're using here/changes you made. I'm having trouble reproducing this error. (Firefox 89/Chrome 91 on mac).

Reply:
🤔  I'm seeing it on Chrome 91, MacOS. It was in Safari too. Here's another image showing the focus state border issue more clearly. If you tab through the page down to the photo tiles, maybe it'll be more clear? 
![Screen Shot 2021-07-10 at 12 03 25 PM](https://user-images.githubusercontent.com/38137359/125171005-7417b280-e177-11eb-93cf-2956756f8021.png)

Reply:
Ah gotcha. I see now, I don't think I was 100% certain what I was looking for lol. yup, that 3.25 looks better!